### PR TITLE
Granularize harvesting skinbits

### LINKED ISSUE #27390: butchery returns seem way off
**Describe the bug**  
Butchery results seem way off.
I got onto this when I realized that butchering wolves and coyotes always fails to produce a pelt for me.
(Also Butchering a zombie yields no tainted hide, which I find strange since butchering an NPC gives you a skin but this may be totally unrelated to this).

I played around with debug mode for a bit and found out that bears give you a pelt.
Wolves never give you a pelt as far as I can tell (N=25)
So I am thinking this may have to do with animal size.

**To Reproduce**  
I have spawned 25 wolves.
All were killed by debug command so there should be no damage to the corpse from excess violence.
I then hauled them onto a butchering rack with an adjacent table.
Then I performed full butchery on all of them.
Character has 20 in all skills, a butchers knife, a tool box, a butchering rack and a table next to the rack.
Attached below you find the results of butchering 25 wolves.

**Expected behavior**  
Wolves should give you something to tan.
I suspect this has something to do with animal size and butchery returns as a black bear does yield a pelt.
But if I look at the screenshot there are more returns that seem way of.
For example only one kidney out of 25 wolves for a 20 survival character with essentially best conditions as far as i understand?

Even if the character fails to harvest an intact wolf pelt I think you should get at least a partial hide.
Realistically even rather small animals like rabbits should give you a small hide that you can tan for a leather patch.
Maybe needs introduction of "small hide" and "small fur" items that way small animals can give some hide returns and for big animals the game could downgrade a big hide to several small hides when the retrieving the hide fails.

**Screenshots**  

**Versions and configuration(please complete the following information):**
 - OS: Ubuntu 18.04
 - Game Version:0.C-35997-g2494eed669 (compiled around 11:00 CET)
 - Graphics version: Tiles
 - Mods loaded: default mods

**Additional context**  
Butchery yields from 25 wolves:
screenshot from 2019-01-01 14-59-04
screenshot from 2019-01-01 15-08-59

PS: Changing the raw_pelt entry in mammal_fur from 0.02 to 0.202 resulted in 198 raw pelts from 25 wolves. This makes me believe the ratios for medium sized animals are just way off.

### PR DESCRIPTION
**Pull request by Toothspit**
#### Summary

```SUMMARY: Bugfixes "Skinning animals smaller than Great Pyrenees now possible"```

#### Purpose of change

Fixes #27390, inability to yield raw_pelt or raw_fur from creatures smaller than 41600g.

#### Describe the solution

With 832g-mass items as the smallest yield when butchering, there is a minimum animal size that will ever possibly yield a raw_fur or raw_pelt item. Decreasing the mass of the yielded item results in more granular yields, and decreases the minimum creature size that could possibly yield a raw_fur or raw_pelt. I chose an ensmallification factor of 50.

That minimum creature to ever yield raw_pelt or raw_fur is now the beagle/dachsund. Seemed reasonable, though we are still unable to skin cats, mink, or puppies due to their being `"harvest": "mammal_tiny"`, but these are just further little JSON tweaks if folks disagree (minimum skinnable animal size is now 832g). ~~Note that adjusting this cutoff makes raw_fur and raw_pelt items take up 10x more space due to the minimum volume of 1mL. This was the compromise I made.~~ Thanks @NotFuji!

#### Describe alternatives you've considered

A factor of 10, or bazillionty. I'm not aware of any historical chipmunk-fur cottage industries, and it doesn't seem reasonable for our survivors to try to salvage a gram or two of fur bits at granularization factors greater than 50. Animals smaller than 10kg are already classified as `mammal_tiny` and yield no skin, though I'm not personally sure that mink and fox should fall in that category; this PR doesn't address that balance aspect, and is a bugfix. Recipe balance has not been changed.

Treating raw_ skin items as a mass, hodgepodge, sewn-together collection similar to liquids is way outside the scope of a bugfix.

Using individual, unique animal pelts in crafting is also way out of scope for a bugfix. Animals smaller than 40kg should be skinnable. I'm just moving the line to .8kg here.

#### Additional context

See issue #27390 or https://www.reddit.com/r/cataclysmdda/comments/abe14z/difficulty_harvesting_pelts_and_hides/ . ~~Note that pelts now take up 10x more volume, I thought that cutoff was worth hitting to fix this way-less-obvious-to-users cutoff.~~ Thanks @NotFuji!

Also sneaky-sneaky copy-from'd tainted fur, I was already there. Edit: tainted leather too.

### CODE REVIEW

**Comment by NotFuji** [comment]
You can define smaller volumes as `"10ml"` or however much you want, quotes included.

**Comment by Toothspit** [comment]
Well that was not at all obvious. One sec, fixing.

**Comment by Toothspit** [comment]
Thanks @NotFuji! Sorry Travis.

File: data/json/recipes/recipe_others.json [hunk 1]
```diff
@@ -3531,7 +3531,7 @@
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [
       [ [ "salt_water", 2 ], [ "saline", 10 ], [ "salt", 10 ] ],
-      [ [ "raw_leather", 1 ], [ "raw_tainted_leather", 1 ], [ "raw_hleather", 1 ] ]
+      [ [ "raw_leather", 1 ], [ "raw_tainted_leather", 50 ], [ "raw_hleather", 50 ] ]
```
Comment (GoLoT) [inline data/json/recipes/recipe_others.json]: Is this supposed to be `[ "raw_leather", 1 ]`?

Comment (Toothspit) [reply data/json/recipes/recipe_others.json]: Nope. Thanks!

**Comment by SeelenJaegerTee** [comment]
Thanks for the hotfix I really appreciate it.

I think the cleanest long term solution would be to introduce "small raw hide/pelt" that turns into "tanning small hide/pelt" and into "leather patch/fur pelt".
That way all animals that fail to produce a big skin could give multiple small skins. If animals that are/were historically used for fur/leather production like house cat, minx or bunnies are too small to yield one of those I propose just giving them a fixed return of 1 small hide.
This would result in no need to fiddle around with current recipes and would solve the problem of skinless cats and bunnies in one go.

**Comment by Toothspit** [comment]
I definitely agree. However, this is a bugfix make-and-merge PR, not meant to tackle any larger issues. The solution you describe is worth its own issue and later PR to implement. I simply saw this reported on Reddit as well as here and wanted to resolve an obtuse issue for players without under-the-hood knowledge of the problem as concisely and quickly as possible.

**Comment by Inglonias** [comment]
What happens to existing fur pelts? I recently grabbed a few off a bear, and I don't want to suddenly discover that the two I have have shrunk by a massive amount.

**Comment by KorGgenT** [comment]
I noticed there is no thread or anything holding all these bits of leather together.

**Comment by Toothspit** [comment]
@Inglonias they are destructively reduced tenfold. Introducing code to grandfather old pelts would be code clutter, IMO. If there are a significant number of users hoarding raw_whatever for the whole 36 hours it lasts, we can close this PR as not worth the loss.

@KorGgenT There isn't. any I wholeheartedly back any PR that wants to tackle the issue, this is simply a bugfix. Wolf pelts should be usable by players that kill and butcher wolves, which use `"harvest": "mammal_fur"`; simple as that for now.

**Comment by Inglonias** [comment]
If this PR only affects the raw pelts, then I think we're fine. You'd lose them soon anyhow, and you can debug them back in if you need to.

**Comment by kevingranade** [comment]
The volume of the pelt harvested from a wolf seems off still, at lvl 10 skill with a butcher knife I only harvested 26 units of hide, that's about half what you need for a single hide bag or a single unit of leather, that seems way off.

That's not what this PR is addressing though so I'm going to go ahead and merge.

**Pull request merged**